src/edu/washington/escience/myria/coordinator/catalog/MasterCatalog.java

```diff
@@ -117,6 +115,7 @@
     + "    program_name TEXT NOT NULL,\n"
     + "    relation_name TEXT NOT NULL,\n"
     + "    num_tuples INTEGER NOT NULL,\n"
+    + "    tombstone INTEGER NOT NULL,\n"
```

Comment:
I think a less jargon-ish and more descriptive name than `tombstone` would be appropriate, maybe `is_deleted` or `deleting` or `marked_for_deletion`?